requirements.txt

```diff
@@ -1,6 +1,7 @@
 Django==2.2
 django-authtools==1.7.0
 djangorestframework==3.12.2
-djangorestframework-simplejwt==4.6.0
+djangorestframework-simplejwt==4.4.0
 dj-rest-auth==2.1.1
 django-allauth==0.42.0
+django-rest-swagger
```

Comment:
What is the version number of this package?

Reply:
django-rest-swagger==2.2.0
requirement.txt is updated with other newly installed softwares.

Reply:
Can't see the updates :) Make sure you've pushed them.